src/components/ScreenStart.vue

```diff
@@ -87,9 +117,44 @@ export default defineComponent({
                 setMessage(message.value.split(matches[0]).join(type.value))
             }
         }
+        const helpData = {
+            type: {
+                title: 'type',
+                url: 'https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md#type',
+                description: 'The type of the work that is being described by this CITATION.cff file.',
+                examples: [
+                    'software',
+                    'dataset'
+                ]
```

Comment:
```suggestion
                description: 'The type of the work that is being described by this CITATION.cff file.'
```
If we make examples optional.